css/editor.css

```diff
@@ -7,6 +27,17 @@ html, body {
     padding: 0;
 }
 
+input, textarea, select, button, .lm_content .dg .c input[type=text], small, .lm_content .dg li.cr,
```

Comment:
My problem is we have to specify eveyrthing individually
Is it not possible to say "everybody uses this font" ?

Reply:
Or possible to say "everybody execpt this one etc." ?